src/patternfly/components/AppLauncher/app-launcher.scss

```diff
@@ -230,6 +234,10 @@
       --pf-c-app-launcher__menu-item--m-action--Color: var(--pf-c-app-launcher__menu-item--m-action--hover--Color);
     }
   }
+
+  &.pf-m-description {
+    display: block;
```

Comment:
Hmm... this is going to cause the styling between non-description and description items to be different. A good example is an item with favorites.

I wonder if this might be a use case for absolutely positioning a pseudo element of `__menu-item` relative to the parent `li` so it covers the entire `li`. Then placing the `__menu-item-description` element after `__menu-item` element. So technically they're different elements, but the pseudo element of the `__menu-item` covers the `__menu-item-description` element, so clicking on anywhere in the `li` triggers a click to the `__menu-item`.

Reply:
Tried this and it works really well from what i can see!